lib/phoenix_live_view/html_engine.ex

```diff
@@ -444,4 +442,13 @@ defmodule Phoenix.LiveView.HTMLEngine do
         raise SyntaxError, line: line, column: column, file: file, description: message
     end
   end
+
+  defp pop_args({wrapper, meta, kw}) do
```

Comment:
I don't think this is going to work if we have dynamic attributes, because then this AST will look like this:

    Map.merge(dynamic, static)

I think it is probably better to pop the args on handle_component_attrs, as we work on the static attributes, instead of doing it later on. WDYT?

Reply:
We should also add tests for the above :)

Reply:
Good catch! I totally missed dynamic attributes.